### ISSUE TITLE
Need to realign Additional Accessibility information examples from Principles and Technics. 

### ISSUE THREAD

**Issue by gautierchomel** [w3c/publ-a11y#392]
We have lost synchronisation between Principles and Technics examples for Additional Accessibility Information. 

### Current examples in Principles : 

- Descriptives
  - Content is enhanced with ARIA roles to optimize organization and facilitate navigation.
  - Page breaks included from the original print source.
  - Videos included in publications have closed captions. 
  - Tactile graphics have been integrated to facilitate access to visual elements for blind people.
  - Music scores are provided that are visually accessible.
  - Content contains information encoded through colors, which may be imperceptible to those who cannot distinguish colors.
- Compacts
  - ARIA roles included.
  - visible page numbering
  - Closed captions for videos.
  - Tactile graphics attached.
  - Music scores visually presented.
  - Information encoded through colors.
 
### Current examples in Technics : 

-   Adaptation
    -  dyslexia readability
    -  closed captions
    -  open captions
    -  full transcript
    -  sign language

- Clarity
  -   text-to-speech hinting provided
  -   color is not the sole means of conveying information
  -   high contrast between text and background
  -   ultra high contrast between text and background
  -   visible page numbering
  -   high contrast between foreground and background audio
  -   without background sounds

**Comment by gautierchomel** [comment]
The proposal is to rewrite principles examples (compact and descriptive) to align with technics.

**Comment by GeorgeKerscher** [comment]
I believe we have resolved this problem with the latest release. If there are still problems, please submit a new issue.
